Linux/tree-common/etc/hosts.tmpl

```diff
@@ -0,0 +1,5 @@
+127.0.1.1       HOSTNAME FQDN
```

Comment:
I believe the FQDN is supposed to come first, otherwise `hostname -f` will show HOSTNAME instead of FQDN.